userland/simple-ble/Makefile

```diff
@@ -0,0 +1,9 @@
+# Base folder definitions
```

Comment:
Add a README to the `simple-ble` folder explaining what the purpose of the library is

Reply:
@alevy I leave this to you :)

Reply:
Done